### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by flamendless** [matiasah/shadows#22]
Hi, are the demo .love files really slow? Because it's really slow in my laptop :/ the fps shown in the standard demo file is 0, texture memory is around 19mb, lua memory is just about 2mb. The mouse movement is pretty delayed :/ I think that's because of the shaders

**Comment by matiasah** [comment]
The demos are especially showing off features based on textures (heightmaps, normalmaps, etc). The calculation of those may be slow. I could make a sample that only shows off rectangular/circular shapes, if that's what you need.

**Comment by flamendless** [comment]
yes, that would really be helpful. also, can you provide a demo for a platformer/sidescroller view?
